geonode_mapstore_client/client/js/plugins/MetadataEditor/components/_fields/SchemaField.jsx

```diff
@@ -104,7 +111,16 @@ const SchemaField = (props) => {
                             })
                         };
                     });
-            }
+            },
+            error: isEmpty(errors) ? null : <ul>
+                {errors.map((error, idx) => {
+                    return (
+                        <li key={idx} className="gn-error-message">
+                            {error}
+                        </li>
+                    );
+                })}
+            </ul>
```

Comment:
We could avoid to use a custom class and use the text-danger instead because it used also by other fields
```js
error: !errors?.length ? null : <>
    {errors.map((entry, idx) => {
        return (
            <React.Fragment key={idx}>
                <ul className="text-danger">
                    {entry.messages.map((message, mIdx) => {
                        return <li key={mIdx}>{entry.key ? `${entry.key} ` : ''}{message}</li>;
                    })}
                </ul>
            </React.Fragment>
        );
    })}
</>
```